esmvaltool/cmorizers/data/datasets.yml

```diff
@@ -0,0 +1,884 @@
+
+  CDS-SATELLITE-SOIL-MOISTURE:
+    tier: 3
+    source: https://cds.climate.copernicus.eu/cdsapp#!/dataset/satellite-soil-moisture?tab=form
+    last_access: 2019-03-14
+    info: ""
+
+  CDS-UERRA:
+    tier: 3
+    source: https://cds.climate.copernicus.eu/cdsapp#!/dataset/reanalysis-uerra-europe-soil-levels
+    last_access: 2019-11-04
+    info: |
+      - Open in a browser the data source as specified above
+      - Put the right ticks:
+        - Tick Origin UERRA-HARMONIE
+        - Tick Variable 'Volumetric soil moisture'
+        - Tick Soil level 1, 2, 3
+        - Tick all available years
+        - Tick all available months
+        - Tick all available days
+
+  CDS-XCH4:
+    tier: 3
+    source: https://cds.climate.copernicus.eu/cdsapp#!/dataset/satellite-methane?tab=form
+    last_access: 2019-03-11
+    info: |
+      Select Processing level "Level 3", variable "Column-average dry-air mole
+      fraction of atmospheric methane (XCH4) and related variables", Sensor and
+      algorithm "MERGED and OBS4MIPS".
+      A registration is required to download the data.
+
+  CDS-XCO2:
+    tier: 3
+    source: https://cds.climate.copernicus.eu/cdsapp#!/dataset/satellite-carbon-dioxide?tab=form
+    last_access: 2019-03-19
+    info: |
+      Select Processing level "Level 3", variable "Column-average dry-air mole
+      fraction of atmospheric carbon dioxide (XCO2) and related variables",
+      Sensor and algorithm "MERGED and OBS4MIPS".
+      A registration is required to download the data.
+
+  CERES-EBAF:
+    tier: 2
+    source: https://ceres-tool.larc.nasa.gov/ord-tool/jsp/EBAF4Selection.jsp
+    last_access: 2019-11-26
+    info: |
+      Select: "TOA Fluxes" ("Shortwave Flux" and "Longwave Flux", "All Sky"
+      and "Clear Sky"), "Monthly Mean", "Regional 1x1 global grid".
+
+  CERES-SYN1deg:
+    tier: 3
+    source: https://ceres-tool.larc.nasa.gov/ord-tool/jsp/SYN1degSelection.jsp
+    last_access: 2019-02-07
+    info: |
+      Monthly data:
+        Expand "Compute TOA Fluxes" and select:
+          Shortwave Flux, Allsky and Clearsky
+          Longwave Flux, Allsky and Clearsky
+          Shortwave Flux Down, Allsky
+        Expand "Computed Surface Fluxes" and select:
+          Shortwave Flux Up, Allsky and Clearsky
+          Shortwave Flux Down, Allsky and Clearsky
+          Longwave Flux Up, Allsky and Clearsky
+          Longwave Flux Down, Allsky and Clearsky
+        then click on "Monthly", "Regional" and "Get data". All fields are saved
+        in CERES_SYN1deg-Month_Terra-Aqua-MODIS_Ed3A_Subset_200003-201702.nc
+      3hr data:
+        Select the same fields as above, then click on "Daily 3-Hourly" and
+        "Get data". All fields are saved in
+        CERES_SYN1deg-3H_Terra-Aqua-MODIS_Ed3A_Subset_YYYYMMDD-YYYYMMDD.nc
+      Put all files in input_dir_path (no subdirectories with years).
+      For orders larger than 2 GB a registration is required.
+
```

Comment:
The dataset `CLARA-AVHRR` is missing from this file. It is not shown in the list generated by `esmvaltool data list`. Besides, its cmorizer script is not working. I think that one was not upgraded using the new framwork.

Reply:
It is added now